lib/Validators/BoardServiceValidator.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * @copyright Copyright (c) 2016 Julius Härtl <jus@bitgrid.net>
+ *
+ * @author Julius Härtl <jus@bitgrid.net>
+ * @author Maxence Lange <maxence@artificial-owl.com>
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+declare(strict_types=1);
+
+namespace OCA\Deck\Validators;
+
+class BoardServiceValidator extends BaseValidator {
+	public function rules() {
+		return [
+			'id' => ['numeric'],
+			'mapper' => ['not_empty', 'not_null', 'not_false'],
+			'title' => ['not_empty', 'not_null', 'not_false', 'max:100', 'min:5'],
```

Comment:
```suggestion
			'title' => ['not_empty', 'not_null', 'not_false', 'max:100'],
```

This is causing the test failures it seems as it is a new restriction